release_management/create_artifacts.py

```diff
@@ -111,9 +118,14 @@ def _stratisd_artifacts(namespace):
 
     r_v = ReleaseVersion(release_version, namespace.pre_release_suffix)
 
+    filtered = bool(namespace.vendor_method == "filtered")
```

Comment:
You can omit the ```bool``` call, since the ```==``` expression evaluates to a bool, anyway.

Reply:
Resolved.